### ISSUE TITLE
Droppable compile errror

### ISSUE THREAD

**Issue by confile** [ArcBees/gwtquery-droppable-plugin#5]
I use

```
<inherits name='gwtquery.plugins.droppable.Droppable'/>
````
After compile I get: 

```
Validating units:
  |       [ERROR] Errors in 'gwtquery/plugins/droppable/client/gwt/DragAndDropCellBrowser.java'
  |          [ERROR] Line 65: The type BrowserCellList is not visible
  |          [ERROR] Line 171: The type CellBrowser.BrowserCellList<T> is not visible
  |          [ERROR] Line 180: The type CellBrowser.BrowserCellList<T> is not visible
  |          [ERROR] Line 197: The type CellBrowser.BrowserCellList<T> is not visible
  |          [ERROR] Line 219: The method getDefaultResources() from the type CellBrowser is not visible
  |          [ERROR] Line 332: The type BrowserCellList is not visible
  |          [ERROR] Line 332: The method createDisplay(TreeViewModel.NodeInfo<C>, int) of type DragAndDropCellBrowser must override or implement a supertype method
  |          [ERROR] Line 336: The method getLoadingIndicator() is undefined for the type DragAndDropCellBrowser
  |          [ERROR] Line 337: The method getLoadingIndicator() is undefined for the type DragAndDropCellBrowser
  |          [ERROR] Line 339: The method getPageSize() is undefined for the type DragAndDropCellBrowser
  |          [ERROR] Line 340: The method getPageSize() is undefined for the type DragAndDropCellBrowser
  |       [ERROR] Errors in 'gwtquery/plugins/droppable/client/gwt/DragAndDropCellTree.java'
  |          [ERROR] Line 43: The type com.google.gwt.user.cellview.client.CellTreeNodeView is not visible
  |          [ERROR] Line 87: CellTreeNodeView cannot be resolved to a type
  |          [ERROR] Line 87: The method createTreeNodeView(T, CellTree.CellTreeMessages) of type DragAndDropCellTree must override or implement a supertype method
  |          [ERROR] Line 88: The constructor DragAndDropCellTreeNodeView<T>(DragAndDropCellTree, CellTreeNodeView<?>, TreeViewModel.NodeInfo<T>, Element, T, CellTree.CellTreeMessages) refers to the missing type CellTreeNodeView
  |       [ERROR] Errors in 'gwtquery/plugins/droppable/client/gwt/DragAndDropCellTreeNodeView.java'
  |          [ERROR] Line 25: The type com.google.gwt.user.cellview.client.CellTreeNodeView is not visible
  |          [ERROR] Line 35: CellTreeNodeView cannot be resolved to a type
  |          [ERROR] Line 46: NodeCellList cannot be resolved to a type
  |          [ERROR] Line 51: NodeCellList cannot be resolved to a type
  |          [ERROR] Line 62: The method replaceAllChildren(List<C>, SelectionModel<? super C>, boolean) of type DragAndDropCellTreeNodeView<T>.DragAndDropNodeCellList<C>.View must override or implement a supertype method
  |          [ERROR] Line 67: The method replaceAllChildren(List<C>, SelectionModel<capture#1-of ? super C>, boolean) is undefined for the type Object
  |          [ERROR] Line 74: The method replaceChildren(List<C>, int, SelectionModel<? super C>, boolean) of type DragAndDropCellTreeNodeView<T>.DragAndDropNodeCellList<C>.View must override or implement a supertype method
  |          [ERROR] Line 83: The method replaceChildren(List<C>, int, SelectionModel<capture#2-of ? super C>, boolean) is undefined for the type Object
  |          [ERROR] Line 98: The method getNodeInfo() is undefined for the type DragAndDropCellTreeNodeView.DragAndDropNodeCellList<C>.View
  |          [ERROR] Line 101: The method getNodeInfo() is undefined for the type DragAndDropCellTreeNodeView.DragAndDropNodeCellList<C>.View
  |          [ERROR] Line 109: The method getNodeView() is undefined for the type DragAndDropCellTreeNodeView.DragAndDropNodeCellList<C>.View
  |          [ERROR] Line 138: CellTreeNodeView cannot be resolved to a type
  |          [ERROR] Line 147: NodeCellList<C> cannot be resolved to a type
  |          [ERROR] Line 152: NodeCellList cannot be resolved to a type
  |          [ERROR] Line 154: The method getNodeView() is undefined for the type DragAndDropCellTreeNodeView.DragAndDropNodeCellList<C>
  |          [ERROR] Line 179: CellTreeNodeView cannot be resolved to a type
  |          [ERROR] Line 186: CellTreeNodeView cannot be resolved to a type
  |          [ERROR] Line 188: The constructor DragAndDropCellTreeNodeView<C>(DragAndDropCellTree, CellTreeNodeView<?>, TreeViewModel.NodeInfo<C>, Element, C, CellTree.CellTreeMessages) refers to the missing type CellTreeNodeView
  |          [ERROR] Line 197: NodeCellList cannot be resolved to a type
  |          [ERROR] Line 197: The method createNodeCellList(TreeViewModel.NodeInfo<C>) of type DragAndDropCellTreeNodeView<T> must override or implement a supertype method
  |          [ERROR] Line 199: The constructor DragAndDropCellTreeNodeView.DragAndDropNodeCellList<C>(TreeViewModel.NodeInfo<C>, CellTreeNodeView<?>, int) refers to the missing type CellTreeNodeView
  |    [ERROR] Aborting compile due to errors in some input files
```

**Comment by opncow** [comment]
I know this one is old but I think that this is because the order of the jars on the classpath is wrong.

This is how it works for me in IntelliJ IDEA:
![gquery](https://cloud.githubusercontent.com/assets/1069651/6610856/7cd29714-c865-11e4-81dc-a53cded52d90.png)

**Comment by jclariviere** [comment]
This should be fixed by #7 .
